src/main/app/src/app/shared/shared.module.ts

```diff
@@ -95,7 +96,8 @@ import {VersionComponent} from './version/version.component';
         MaterialFormBuilderModule.forRoot(),
         TranslateModule,
         PdfJsViewerModule,
-        NgxSkeletonLoaderModule.forRoot({animation: 'pulse'})
+        NgxSkeletonLoaderModule.forRoot({animation: 'pulse'}),
+        DocumentIconModule
```

Comment:
Het is niet nodig om de DocumentIconModule te importeren in de SharedModule, daar wordt hij niet gebruikt. Hij wordt gebruikt in de MaterialFormBuilderModule waar hij als losse module geimporteerd wordt en verder op andere plekken, waar hij via de SharedModule meekomt.

Reply:
Hij wordt gebruikt in de templates van de `zaak-documenten`, `informatie-object-view` en `documenten-lijst`-componenten. Alleen bij de laatste daarvan komt hij al binnen via de import in de MaterialFormBuilderModule, maar bij die andere twee niet.

Reply:
Verwijderd uit de imports